lib/dash/dash_parser.js

```diff
@@ -768,34 +768,37 @@ shaka.dash.DashParser = class {
     for (const prop of accessibilities) {
       const schemeId = prop.getAttribute('schemeIdUri');
       const value = prop.getAttribute('value');
-      if (schemeId == 'urn:scte:dash:cc:cea-608:2015' ||
-          schemeId == 'urn:scte:dash:cc:cea-708:2015') {
+      if (schemeId == 'urn:scte:dash:cc:cea-608:2015' ) {
         let channelId = 1;
         if (value != null) {
-          for (const captionStr of value.split(';')) {
+          const channelAssignments = value.split(';');
+          for (const captionStr of channelAssignments) {
             let channel;
             let language;
             // Some closed caption descriptions have channel number and
-            // language, like "CC1=eng" or "1=lang:eng", others may only have
-            // the language, like "eng".
+            // language ("CC1=eng") others may only have language ("eng,spa").
             if (!captionStr.includes('=')) {
-              // Since only odd numbers are used as channel numbers, like CC1,
-              // CC3, CC5, etc, when the channel number is not provided, use an
-              // odd number as the key. https://en.wikipedia.org/wiki/EIA-608
-              channel = 'CC' + channelId;
-              channelId += 2;
+              // When the channel assignemnts are not explicitly provided and
+              // there are only 2 values provided, it is highly likely that the
+              // assignments are CC1 and CC3 (most commonly used CC streams).
+              // Otherwise, cycle through all channels arbitrarily (CC1 - CC4)
+              // in order of provided langs.
```

Comment:
This case does not seem to be covered by the tests.  Can you please add one more?